sdk/servicebus/service-bus/src/models.ts

```diff
@@ -152,12 +152,16 @@ export interface GetMessageIteratorOptions extends OperationOptionsBase {}
  */
 export interface SubscribeOptions extends OperationOptionsBase {
   /**
-   * @property Indicates whether the `complete()` method on the message should automatically be
-   * called by the sdk after the user provided onMessage handler has been executed.
-   * Calling `complete()` on a message removes it from the Queue/Subscription.
+   * @property Indicates whether the message should be settled using the `completeMessage()`
+   * method on the receiver automatically after it executes the user provided message callback.
```

Comment:
```suggestion
   * method on the receiver automatically after it executes the user-provided `processMessage` callback.
```

Reply:
?

Reply:
I didnt want to use `processMessage` and instead wanted to stick to equivalent english words